packages/flutter_tools/test/general.shard/fuchsia/fuchsia_kernel_compiler_test.dart

```diff
@@ -0,0 +1,52 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_tools/src/build_info.dart';
+import 'package:flutter_tools/src/fuchsia/fuchsia_kernel_compiler.dart';
+
+import '../../src/common.dart';
+
+void main() {
+  group('Fuchsia Kernel Compiler', () {
```

Comment:
Prefer 2 space indents in this file

Reply:
I don't understand this one, what should I do?